src/store/reducers/accounts.ts

```diff
@@ -475,6 +517,7 @@ export default (state: AccountsState = initialState, action): AccountsState => {
           service: action.payload.services[SECURE_ACCOUNT],
         },
         servicesEnriched: true,
+        accountShells: updatedAccountShells,
```

Comment:
If the idea is that we can't really create account shells until `SERVICES_ENRICHED` occurs, perhaps it would be best to set `accountShells` to an empty array in the initial state.

Reply:
Yes, have made that change in the latest commit